### PR TITLE
C#intermediate tutorial - improvements

### PR DESCRIPTION
**Pull request by Aggror**
# PR Details
Various small improvements to the intermediate C# tutorials whilst making thumbnails and recording videos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,22 @@
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 =======
 
+
+[![Financial sponsors](https://img.shields.io/opencollective/all/stride3d)](https://opencollective.com/stride3d)
+
+[![Join the chat at https://discord.gg/f6aerfE](https://img.shields.io/discord/500285081265635328.svg?style=flat&logo=discord&label=discord)](https://discord.gg/f6aerfE)
```
Comment (Eideren) [inline README.md:12]: Looks like you didn't mean to copy this one, it's already listed above

File: README.md [hunk 2]
```diff
@@ -6,6 +6,22 @@
...
+
+
+
+
+
+<img alt="open collective badge" src="https://opencollective.com/stride3d/tiers/copper-strider/badge.svg&color=brightgreen" />
+
+https://opencollective.com/stride3d/tiers/badge.svg
+
+<object type="image/svg+xml" data="https://opencollective.com/stride3d/tiers/copper-strider.svg?avatarHeight=36&width=600"></object>
```
Comment (Eideren) [inline README.md:22]: Those links don't seem to display as expected

File: samples/Tutorials/CSharpIntermediate/CSharpIntermediate/CSharpIntermediate.Game/02_Collision-Triggers/CollisionTriggerDemo.cs [hunk 3]
```diff
@@ -10,47 +10,36 @@ namespace CSharpIntermediate.Code
...
         {
-            // The trigger collider can have 0, 1, or multiple collision going on in a single frame
-            var drawY = 280;
-            foreach (var collision in triggerCollider.Collisions)
+            foreach (var collision in staticCollider.Collisions)
             {
-                DebugText.Print("ColliderA: " + collision.ColliderA.Entity.Name, new Int2(500, drawY += 20));
-                DebugText.Print("ColliderB: " + collision.ColliderB.Entity.Name, new Int2(500, drawY += 20));
+                DebugText.Print("ColliderA: " + collision.ColliderA.Entity.Name, new Int2(500, 300));
+                DebugText.Print("ColliderB: " + collision.ColliderB.Entity.Name, new Int2(500, 320));
```
Comment (Eideren) [inline samples/Tutorials/CSharpIntermediate/CSharpIntermediate/CSharpIntermediate.Game/02_Collision-Triggers/CollisionTriggerDemo.cs:42]: Did you mean to remove the draw variable ?

**Review by Eideren — COMMENTED** [review]
Thanks ! 'Took a quick look

**Comment by Aggror** [comment]
Hi @Eideren,

thank you for taking the time to have a look. Yeah so for the recording I create a new tutorial project with the intermediate tutorials. But looks I fixed/ improvement the scenes/code in that new project instead of the actual sample template.

**Pull request merged**